lucille-core/src/main/java/com/kmwllc/lucille/connector/jdbc/DatabaseConnector.java

```diff
@@ -199,24 +201,30 @@ public void execute(Publisher publisher) throws ConnectorException {
         publisher.publish(doc);
       }
 
-      // close all results
-      rs.close();
-      for (ResultSet ors : otherResults) {
-        ors.close();
-      }
-      // the post sql.
-      runSql(connection, postSql);
-      flush();
+
     } catch (Exception e) {
       throw new ConnectorException("Exception caught during connector execution", e);
+    } finally {
+      // we are closing the result sets
+      if (rs != null) {
+        try {
+          rs.close();
+        } catch (SQLException e) {
+          log.warn("Result set unable to be closed");
+        }
+      }
+      if (otherResults != null) {
+        for (ResultSet ors : otherResults) {
+          try {
+            ors.close();

```

Comment:
need to null check each ors before calling close

Reply:
Added null checking in new commit